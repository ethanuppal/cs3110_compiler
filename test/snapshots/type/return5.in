func main() -> Int {
    // last statement here is terminal, since there is no 'else' branch we don't have a problem saying it halts
    if true {
        return 0
    }
}
