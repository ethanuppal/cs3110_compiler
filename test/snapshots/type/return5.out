func main() -> Int {
    if true {
        return 0
    }
}
